heliopy/data/sunspot.py

```diff
@@ -1,3 +1,13 @@
+"""
+Sunspot
+-------
+
+Methods for automatically downloading sunspot number data.
+
+For more info about the sunspot number data, visit
+http://www.sidc.be/silso/infosndtot.

```

Comment:
"http://www.sidc.be/silso/infosndtot" redirects to Daily Sunspot, should it be changed to "http://www.sidc.be/silso/datafiles" which redirects to the main page? We already refer to the Daily Sunspot Number in its own section

Reply:
Yeah, that would be good, thanks for spotting that!